Public/Src/Engine/Scheduler/PipExecutor.cs

```diff
@@ -646,6 +646,15 @@ private static PipResultStatus FromPossibleResult(Possible<PipResultStatus> poss
                         connectionString,
                         ipcResult.Payload);
                 }
+                else if (ipcResult.ExitCode == IpcResultStatus.TransmissionError)
+                {
+                    // we separate transmission errors here, so they can be properly classified as InfrastructureErrors
+                    Logger.Log.PipIpcFailedDueToInfrastructureError(
+                        operationContext,
+                        operation.Payload,
+                        connectionString,
+                        ipcResult.Payload);
+                }
                 else
                 {
                     Logger.Log.PipIpcFailed(

```

Comment:
>PipIpcFailed [](start = 31, length = 12)

So is this catchall still an internal error? I was expecting to see you add more user error patterns to account for the various user errors we had previously been misclassifying as internal errors.

Reply:
Nevermind. I see you changed the classification in DropDaemon.cs to route more to InvalidInput